src/main/java/graphql/nadel/GraphQLRemoteSchemaSourceFactory.java

```diff
@@ -0,0 +1,35 @@
+package graphql.nadel;
+
+import com.atlassian.braid.Link;
+import com.atlassian.braid.SchemaNamespace;
+import com.atlassian.braid.SchemaSource;
+import com.atlassian.braid.document.DocumentMappers;
+import com.atlassian.braid.source.GraphQLRemoteSchemaSource;
+import graphql.nadel.dsl.ServiceDefinition;
+import graphql.schema.idl.TypeDefinitionRegistry;
+
+import java.util.List;
+
+/**
+ * Schema source factory that uses remote graphql endpoint.
+ *
+ * @param <C> type of passed graphql context.
+ */
+public class GraphQLRemoteSchemaSourceFactory<C> implements SchemaSourceFactory {
```

Comment:
same as the other interface: Braid impl details

Reply:
will make it package private, it is only needed for backward compatibility with old Nadel constructor